minecraft/java/paper/egg-paper.json

```diff
@@ -8,11 +8,12 @@
     "author": "parker@pterodactyl.io",
     "description": "High performance Spigot fork that aims to fix gameplay and mechanics inconsistencies.",
     "features": ["eula"],
-    "image": "quay.io\/pterodactyl\/core:java-11",
+    "image": "quay.io\/parkervcp\/pterodactyl-images:debian_openjdk-16",
```

Comment:
Defaulting to 16 may break current servers. I recommend leaving this at 11 until the new Minecraft is it.